.github/workflows/Tests.yml

```diff
@@ -6,6 +6,24 @@ on:
       - stable
     tags:
       - v*
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - ready_for_review
+    branches-ignore:
+      - '*-patc*'
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - ready_for_review
+    branches:
+      - master
+      - stable
```

Comment:
These changes caused no new issues and should allow for branch rules to be satisfied by most PRs.